### ISSUE TITLE
Wrong null hypothesis indication for homoskedasticity tests

### ISSUE THREAD

**Issue by istar410** [ericqu/LinearRegressionKit.jl#9]
Hello,

First thank you for this amazing package. However I have noticed a little mistake. For the White an Breush-Pagan tests it is indicated in brackets that the null hypothesis is heteroskedasticity. But it is false, both of them test the null hypothesis that residuals are homoskedastic against the alternate hypothesis that they are heteroskedastic.

**Comment by ericqu** [comment]
@istar410 thanks for pointing this out. It should be straightforward to fix this. I am away on holiday for a few weeks, so it won't be immediate sorry for that.
Thank you for using the package, feel free to let me know if there are features you need (no promises that I will implement them but I would like to know a little more about what users are expecting for such a package).

**Comment by istar410** [comment]
No problem, take your time 😄 

As for features, I found your packages very complete. It provides a ton of statistics and tests which are really helpful. So there are not many features that I found were missing. But here are some ideas of little improvements that could be nice to implement (they are mainly based on what the R language proposed):
- add significant codes
- add the F-statistic test
- add descriptive statistics for the residuals
- give the index of influencials individuals and outliers directly on the graphes
